Marlin/language_fr.h

```diff
@@ -189,7 +192,6 @@
 #if LCD_WIDTH > 19
   #define MSG_INFO_PRINT_COUNT              "Nbre impressions"
   #define MSG_INFO_COMPLETED_PRINTS         "Terminees"
-  #define MSG_INFO_PRINT_TIME               "Tps impr. total"
```

Comment:
my bad !
I want to correct this, but this line is still here on my local repo and my git remote... Don't know exactly what to do.
(git noob spotted)